docs/05-rfcs/2023-01-11-sdk-spec.md

```diff
@@ -195,6 +206,7 @@ struct ThresholdReachedEvent {
 * Stream (P2) - a stream of events, similar to AWS Kinesis, Azure Event Hubs, GCP Pub/Sub and Dataflow
 * OnDeploy (P2) - a variation of Function that runs every time the app is deployed
 * GraphQLApi (P2) - a GraphQL API, similar to AWS AppSync
+* SqlDatabase (P2) - a relational database that lets you execute arbitrary SQL queries, similar to AWS RDS, Azure SQL Database, GCP Cloud SQL
```

Comment:
wondering if `Table` above should be `SqlTable` instead to follow this naming scheme? or the other way around, i.e., making it simply `Database`